### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by kevinstratford**
And I have taken the liberty of shortening a rather long file name

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -106,6 +106,5 @@ The easiest way to learn what files should look like is to read the ``rst`` file
 A short list of style guidance:
 
 * Headings should be in sentance case
-* If using ``rst`` tables, use "simple" tables; "grid" tables
-(markup with ``+`` in corners) generate HTML with accessability problems.
+* If using ``rst`` tables, use "simple" tables; "grid" tables (markup with ``+`` in corners) generate HTML with accessability problems.
```
Comment (juanfrh) [inline README.rst:109]: The word "accessability" is mispelled. It should be "accessibility".

**Review by juanfrh — CHANGES_REQUESTED** [review]
Typo found.

File: README.rst (hunk 1 above)
Comment (kevinstratford) [reply README.rst]: Done.

**Review by kevinstratford — COMMENTED** [review]
Should be ok.

**Review by juanfrh — APPROVED** [review]
Typo fixed.

**Pull request merged**